tensorboard/plugins/hparams/list_session_groups.py

```diff
@@ -154,9 +179,18 @@ def _session_groups_from_data_provider(self):
 
             session_groups.append(session_group)
 
+        # Compute the session group's aggregated metrics for each group.
+        for group in session_groups:
+            if group.sessions:
+                # We sort the sessions in a group so that the order is deterministic.
```

Comment:
Will this be moved to the backend (or backer-end) in the future?

Reply:
Now that you mention it, I guess we can already rely on the order returned by the backend, even though theoretically the DataProvider doesn't guarantee any order. In practice this sort is redundant for our use cases.

Anyway, I removed the sorting here (for now).